packages/@vuepress/plugin-pwa/lib/enhanceAppFile.js

```diff
@@ -16,6 +16,7 @@ export default ({ router, isServer }) => {
       && !isServer
       && SW_ENABLED) {
       register(`${SW_BASE_URL}service-worker.js`, {
+        registrationOptions: { scope: `./${SW_BASE_URL}` },
```

Comment:
I'm not sure if this is correct, because `SW_BASE_URL` is an absolute path?

Reply:
I think an empty object for `registrationOptions` is good enough. The default value of `scope` option is the directory that `service-worker.js` file exists.

Reply:
You're right 👍